webforces/server/tests.py

```diff
@@ -9,3 +10,77 @@ def setUp(self):
     def test_core_is_proper_singletone(self):
         core2 = Core()
         self.assertIs(core2, self.core)
+
+
+class DBTest(TestCase):
+    def setUp(self):
+        self.core = Core()
+        self.core.db.dropAll()
+
+    def test_can_add_diff_users(self):
+        self.core = Core()
```

Comment:
You can remove core init in the beginning of every test since we have `setUp()`. `self.core.db.dropAll()` can be extracted to `def tearDown(self):`

More about setup and teardown: https://docs.python.org/3/library/unittest.html#organizing-test-code

Reply:
Thank you! Changed.